src/ui/components/LoadingText/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import "~amo/css/inc/vars";
+
+@keyframes placeHolderShimmer {
+  0% {
+    opacity: 0.5;
+  }
+
+  25% {
+    opacity: 0.15;
+  }
+
+  50% {
+    opacity: 0.75;
+  }
+
+  75% {
+    opacity: 0.15;
+  }
+
+  100% {
+    opacity: 0.5;
+  }
+}
+
+.LoadingText {
+  animation: placeHolderShimmer 3s infinite cubic-bezier(0.65, 0.05, 0.36, 1);
+  background: $text-color-default;
+  display: inline-block;
+  height: 1rem;
+  line-height: 1;
+  margin: 0;
+  vertical-align: middle;
+}
+
+@for $percentage from 1 through 100 {
```

Comment:
This looks like it will add 300 lines to our CSS bundle because of the duplicated rules. Can that be avoided? Could it use a CSS transition or something instead? Or could it do a gradual fade-in like a fake progress indicator?